src/main/server/enable_ssl.sh

```diff
@@ -325,14 +351,12 @@ generate_password(){
 }
 
 check_domain_name(){
-    #check domain name exists
     if [ -z "$domain" ]; then
-    echo "Missing parameter. Domain name is not set"
-    usage
-    exit 1
+      freedomain
```

Comment:
It seems that it does not work If the user specifies his/her own domain name

Reply:
False alarm :)